<launch>
    <!-- Autogenerated launch file - do not hand edit -->
    <include file="/home/albert/ros/mct_config/machine/mct.machine" />
    
    <group ns="/mct_master/camera_8/camera">
        <node pkg="mct_homography" type="homography_calibrator.py" name="homography_calibrator" args="/mct_master/camera_8/camera/image_rect" >
            <rosparam command="load" file="/home/albert/ros/mct_config/tracking_2d/homographies/calibrator_params.yaml"/>
        </node>
    </group>
    
    <group ns="/mct_slave1/camera_4/camera">
        <node pkg="mct_homography" type="homography_calibrator.py" name="homography_calibrator" args="/mct_slave1/camera_4/camera/image_rect" >
            <rosparam command="load" file="/home/albert/ros/mct_config/tracking_2d/homographies/calibrator_params.yaml"/>
        </node>
    </group>
    
    <group ns="/mct_slave1/camera_9/camera">
        <node pkg="mct_homography" type="homography_calibrator.py" name="homography_calibrator" args="/mct_slave1/camera_9/camera/image_rect" >
            <rosparam command="load" file="/home/albert/ros/mct_config/tracking_2d/homographies/calibrator_params.yaml"/>
        </node>
    </group>
    
    <group ns="/mct_slave2/camera_7/camera">
        <node pkg="mct_homography" type="homography_calibrator.py" name="homography_calibrator" args="/mct_slave2/camera_7/camera/image_rect" >
            <rosparam command="load" file="/home/albert/ros/mct_config/tracking_2d/homographies/calibrator_params.yaml"/>
        </node>
    </group>
    
    <group ns="/mct_slave1/camera_2/camera">
        <node pkg="mct_homography" type="homography_calibrator.py" name="homography_calibrator" args="/mct_slave1/camera_2/camera/image_rect" >
            <rosparam command="load" file="/home/albert/ros/mct_config/tracking_2d/homographies/calibrator_params.yaml"/>
        </node>
    </group>
    
    <group ns="/mct_master/camera_3/camera">
        <node pkg="mct_homography" type="homography_calibrator.py" name="homography_calibrator" args="/mct_master/camera_3/camera/image_rect" >
            <rosparam command="load" file="/home/albert/ros/mct_config/tracking_2d/homographies/calibrator_params.yaml"/>
        </node>
    </group>
    
    <group ns="/mct_slave1/camera_1/camera">
        <node pkg="mct_homography" type="homography_calibrator.py" name="homography_calibrator" args="/mct_slave1/camera_1/camera/image_rect" >
            <rosparam command="load" file="/home/albert/ros/mct_config/tracking_2d/homographies/calibrator_params.yaml"/>
        </node>
    </group>
    
    <group ns="/mct_master/camera_11/camera">
        <node pkg="mct_homography" type="homography_calibrator.py" name="homography_calibrator" args="/mct_master/camera_11/camera/image_rect" >
            <rosparam command="load" file="/home/albert/ros/mct_config/tracking_2d/homographies/calibrator_params.yaml"/>
        </node>
    </group>
    
    <group ns="/mct_slave2/camera_5/camera">
        <node pkg="mct_homography" type="homography_calibrator.py" name="homography_calibrator" args="/mct_slave2/camera_5/camera/image_rect" >
            <rosparam command="load" file="/home/albert/ros/mct_config/tracking_2d/homographies/calibrator_params.yaml"/>
        </node>
    </group>
    
    <group ns="/mct_slave2/camera_10/camera">
        <node pkg="mct_homography" type="homography_calibrator.py" name="homography_calibrator" args="/mct_slave2/camera_10/camera/image_rect" >
            <rosparam command="load" file="/home/albert/ros/mct_config/tracking_2d/homographies/calibrator_params.yaml"/>
        </node>
    </group>
     
</launch>